src/preferences/dialog/dlgpreferences.cpp

```diff
@@ -236,6 +236,10 @@ DlgPreferences::~DlgPreferences() {
                        m_geometry.join(","));
     }
 
+    // When DlgPrefControllers is deleted it manually deletes the controller tree items,
+    // which makes QTreeWidgetItem trigger this signal.
```

Comment:
This also fixes the issue. I think the comment should also explain that there would be a segfault without this.